### PR TITLE
Update Fields requests, add /operations

### PR DESCRIPTION
**Pull request by ramnasidharta**
(sorry for the tabulation things, but versioning postman collection files sucks, I think it doesn't worth to lose much time on this. I tried to not commiting that changes on auto generated ids, for a clean PR).

### CODE REVIEW

**Comment by CarlosSam** [comment]
[[Docs] Add in postman: organizationId in Fields entity. Queries by organizationId in /fields. Endpoints fields/{fieldId}/operations[/{id}]](https://trello.com/c/Ke5A22Xq/274-docs-add-in-postman-organizationid-in-fields-entity-queries-by-organizationid-in-fields-endpoints-fields-fieldid-operations-id)

File: LEAF_API.postman_collection.json [hunk 1]
```diff
@@ -1577,6 +1577,122 @@
...
+                                                        {
+                                                                "key": "operationProvider",
+                                                                "value": "",
+                                                                "description": "The name of the company provider of the operation files. Supported values: ClimateFieldView, CNHI, JohnDeere, Trimble",
+                                                                "disabled": true
+                                                        },
+                                                        {
+                                                                "key": "operationStart",
+                                                                "value": null,
+                                                                "description": "ISO 8601 date-time instant when the operation started (must be in the past or present)",
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:1629]: should say that filters for operation that started after or at the same time the parameter instant

File: LEAF_API.postman_collection.json [hunk 2]
```diff
@@ -1577,6 +1577,122 @@
...
+                                                                "disabled": true
+                                                        },
+                                                        {
+                                                                "key": "operationEnd",
+                                                                "value": "",
+                                                                "description": "ISO 8601 date-time instant when the operation ended (must be in the past or present)",
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:1635]: should say that filters for operation that ended before or at the same time the parameter instant

File: LEAF_API.postman_collection.json [hunk 3]
```diff
@@ -1577,6 +1577,122 @@
...
+                                                                "value": "",
+                                                                "description": "Provider crop code/number of the operation. Requires `operationProvider`",
+                                                                "disabled": true
+                                                        },
+                                                        {
+                                                                "key": "operationVariety",
+                                                                "value": "",
+                                                                "description": "Name of the crop variety used by the provider. Requires `operationProvider`. (e.g. \"Corn\" for \"JohnDeere\").",
+                                                                "disabled": true
+                                                        }
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:1649]: missing size, page and sort parameters

File: LEAF_API.postman_collection.json [hunk 4]
```diff
@@ -1577,6 +1577,122 @@
...
+                                                ]
+                                        },
+                                        "description": "Retrieve all operation files from a field.\n\nChange the value after `fields/` in the URL with the ID of the field which you want its files."
+                                },
+                                "response": []
+                        },
+                        {
+                                "name": "Get a field operation",
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:1657]: actually get operations from a single field
also it is missing get a single operation from a given field

File: LEAF_API.postman_collection.json [hunk 5]
```diff
@@ -2247,33 +2253,45 @@
...
+									"value": "",
+									"description": "Supported values: APPLIED, HARVESTED, PLANTED",
+                  "disabled": true
 								},
 								{
 									"key": "operationStart",
-									"value": "1990-01-04T21:18:19.561Z",
-									"description": "ISO date-time instant when the operation started (must be in the past)"
+									"value": "",
+									"description": "ISO 8601 date-time instant when the operation started (must be in the past)",
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:2263]: same thing above, this filters by operations that started after the parameter

File: LEAF_API.postman_collection.json [hunk 6]
```diff
@@ -2247,33 +2253,45 @@
...
+                  "disabled": true
 								},
 								{
 									"key": "operationEnd",
-									"value": "2020-06-01T21:18:19.561Z",
-									"description": "ISO date-time instant when the operation ended (must be in the past or present)"
+									"value": "",
+									"description": "ISO 8601 date-time instant when the operation ended (must be in the past or present)",
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:2269]: same thing above, this filters by operations that ended before the parameter

File: LEAF_API.postman_collection.json [hunk 7]
```diff
@@ -2247,33 +2253,45 @@
...
-									"description": "PascalCase provider name. Supported values: ClimateFieldView, CNHI, JohnDeere, Trimble"
+									"value": "",
+									"description": "The name of the company provider of the operations. Supported values: ClimateFieldView, CNHI, JohnDeere, Trimble",
+                  "disabled": true
+								},
+								{
+									"key": "organizationId",
+									"value": "",
+									"description": "Grower's organization id from the provider (string)",
+                  "disabled": true
```
Comment (CarlosSam) [inline LEAF_API.postman_collection.json:2294]: also missing page, size and sort parameters

File: LEAF_API.postman_collection.json (hunk 4 above)
Comment (ramnasidharta) [reply LEAF_API.postman_collection.json:1657]: > actually get operations from a single field
Actually there is the other endpoint for that ("Get field operations (with filters)"). This one shall be really get a field operation. I wrote in the description of the request:

"Retrieve a specific operation file from a field.

Change the value after fields/ in the URL with the ID of the field related to the operation. Also, put the ID of the operation file after operations/.
"

Do you have any suggestion for making that better?

>  also it is missing get a single operation from a given field
As the above answer, this PR created a folder "Operations" within "4 - Fields". I included two requests in that folder: "Get a field operation" and "Get field operations (with filters)".

**Pull request merged**